### PR TITLE
Fix rubocop

### PR DESCRIPTION
**Pull request by shime**
This pull request fixes some Rubocop issues I encountered when running

```
$ rubocop
```

from the root directory.

### CODE REVIEW

File: guides/rails_guides/markdown/renderer.rb [hunk 1]
```diff
@@ -101,8 +101,7 @@ def github_file_url(file_path)
                    "#{root.sub("_", "")}/lib/#{file_path}"
                  else
                    file_path
-                 end
-
+          end
```
Comment (matthewd) [inline guides/rails_guides/markdown/renderer.rb]: This seems pretty terrible. Which rubocop setting thinks this is a good idea? Maybe we just need to change its mind.

Comment (shime) [inline guides/rails_guides/markdown/renderer.rb]: Yeah, I agree. The metric that is failing is called `Lint/EndAlignment`, so maybe we should just disable that for this method?

**Comment by shime** [comment]
For some reason Code Climate is still reporting a lot of Rubocop issues.

File: guides/rails_guides/markdown/renderer.rb (hunk 1 above)
Comment (shime) [inline guides/rails_guides/markdown/renderer.rb]: I've now disabled that Rubocop check for this method.

File: guides/rails_guides/markdown/renderer.rb [hunk 2]
```diff
@@ -90,6 +90,7 @@ def convert_notes(body)
           end
         end
 
+        # rubocop:disable Lint/EndAlignment
```
Comment (kaspth) [inline guides/rails_guides/markdown/renderer.rb]: We don't want to start sprinkling these into our code base. The only allowed action is to fix our  rubocop rules.

Comment (shime) [inline guides/rails_guides/markdown/renderer.rb]: How do we fix Rubocop rules? The only option is to exclude the file in `.rubocop.yml`, isn't it?

Comment (rafaelfranca) [inline guides/rails_guides/markdown/renderer.rb]: The fix to to change the code to follow what the rule is expecting.

**Review by rafaelfranca — CHANGES_REQUESTED** [review]

File: guides/rails_guides/markdown/renderer.rb (hunk 2 above)
Comment (matthewd) [inline guides/rails_guides/markdown/renderer.rb]: @rafaelfranca https://github.com/shime/rails/commit/082b1fa61a5b880598fde854de559cd98d7926e4 ?!? :confused:

Comment (kaspth) [inline guides/rails_guides/markdown/renderer.rb]: We could just follow the css_class variable assignment above. E.g. `path = \`

Comment (matthewd) [inline guides/rails_guides/markdown/renderer.rb]: Sure.. but if rubocop is insisting on something dumb, we should use its config to tell it to not do that (and hopefully, to instead insist on something less eye-bleedingly wrong).

Using a tool to enforce arbitrary style choices is one thing; having it impose secret work-arounds ("oh, it does that, but you can ignore it, and just do this instead..") is quite another.

Comment (kaspth) [inline guides/rails_guides/markdown/renderer.rb]: Yes, of course 👌

Comment (rafaelfranca) [inline guides/rails_guides/markdown/renderer.rb]: Either way, this is not what this comment is doing right now.

Comment (kaspth) [inline guides/rails_guides/markdown/renderer.rb]: @shime interested in fixing the Rubocop rule? 😊

Comment (shime) [inline guides/rails_guides/markdown/renderer.rb]: @kaspth I'll try to find some time to do it, thanks for the feedback

**Pull request closed without merging**